app/views/splash/index.html.erb

```diff
@@ -1,7 +1,7 @@
-<div class="flex background flex-col items-center bg-blue-400 sm:text-center h-screen justify-center">
-  <h1 class="text-white splash-text my-2">Welcome to costino!</h1>
-
-  <a href="<%= new_user_session_path() %>" class="w-screen my-4 bg-white hover:bg-blue-500 text-blue-400 font-semibold hover:text-white py-2 px-4 block border rounded">
-    <button class=" focus:outline-none">sign in</button>
-  </a>
-</div>
+<div class="flex background flex-col items-center bg-blue-400 sm:text-center h-screen justify-center">
+  <h1 class="text-white splash-text my-2">Welcome to costino!</h1>
+
+  <a href="<%= new_user_session_path() %>" class="w-screen flex justify-center my-4 bg-white hover:bg-blue-500 text-blue-400 font-semibold hover:text-white py-2 px-4 border rounded">
+    <button class=" focus:outline-none">sign in</button>
+  </a>
+</div>
```

Comment:
- [ ] Good job working on the splash screen , but it doesn't quite match the original design. Please kindly ensure it has both the `Sign Up` and `Login` buttons.

### Your Design
![image](https://github.com/mohisa302/budget-app/assets/43995830/a8370a00-33e5-43ae-ae8b-3f9857a552ab)

### Original Design
![image](https://github.com/mohisa302/budget-app/assets/43995830/71553242-280b-48b6-9652-9eeea9268233)

Reply:
Thanks i will implement the changes